config/tipoff.php

```diff
@@ -86,6 +88,10 @@
 
         'place' => \Tipoff\Seo\Models\Place::class,
 
+        'place_details' => \Tipoff\Seo\Models\PlaceDetails::class,
+
+        'place_hours' => \Tipoff\Seo\Models\PlaceHours::class,
```

Comment:
We didn't include `gmb_details` and `gmb_hours` from tipoff/locations because they wouldn't be used by other packages, but I think these will be used by other packages, so I'll leave them.